common/views/components/VideoEmbed/VideoEmbed.tsx

```diff
@@ -76,37 +81,52 @@ const VideoEmbed: FunctionComponent<Props> = ({
     firstScript.parentNode?.insertBefore(s, firstScript);
   }, []);
 
+  // TODO figure out how to not move the whole featured section...
   return (
-    <VideoEmbedWrapper>
-      <IframeContainer>
-        {isActive ? (
-          <iframe
-            className="iframe"
-            title="Video"
-            allowFullScreen={true}
-            allow="autoplay; picture-in-picture"
-            src={`${embedUrl}&enablejsapi=1&autoplay=1`}
-            frameBorder="0"
-          />
-        ) : (
-          <VideoTrigger
-            onClick={() => setIsActive(true)}
-            hasFullSizePoster={hasFullSizePoster}
-          >
-            <span className="visually-hidden">Play video</span>
-            <YouTubePlay />
-            <img
-              src={`https://img.youtube.com/vi/${id}/${
-                hasFullSizePoster ? 'maxresdefault' : 'hqdefault'
-              }.jpg`}
-              alt=""
+    <>
+      <VideoEmbedWrapper>
+        <IframeContainer>
+          {isActive ? (
+            <iframe
+              className="iframe"
+              title="Video"
+              allowFullScreen={true}
+              allow="autoplay; picture-in-picture"
+              src={`${embedUrl}&enablejsapi=1&autoplay=1`}
+              frameBorder="0"
             />
-          </VideoTrigger>
-        )}
-      </IframeContainer>
+          ) : (
+            <VideoTrigger
+              onClick={() => setIsActive(true)}
+              hasFullSizePoster={hasFullSizePoster}
+            >
+              <span className="visually-hidden">Play video</span>
+              <YouTubePlay />
+              <img
+                src={`https://img.youtube.com/vi/${id}/${
+                  hasFullSizePoster ? 'maxresdefault' : 'hqdefault'
+                }.jpg`}
+                alt=""
+              />
+            </VideoTrigger>
+          )}
+        </IframeContainer>
+
+        {caption && <Caption caption={caption} />}
+      </VideoEmbedWrapper>
 
-      {caption && <Caption caption={caption} />}
-    </VideoEmbedWrapper>
+      {transcript && (
+        <CollapsibleContent
+          id={`embedVideoTranscript-${Math.floor(Math.random() * 1000) + 1}`}
```

Comment:
If the VideoEmbed had an id prop you could pass it in from the body with the index of the filteredBody.map it has?

Reply:
I thought I might but it's a URL, so it'd have to be with a regex and as it can be vimeo or youtube, it feels like as much of a hack. 

Maybe just the index would do though! I'll have a look at that.

Reply:
Done here, what do you think? https://github.com/wellcomecollection/wellcomecollection.org/pull/10199/commits/58f2c4e1d836dd001607cb4cefae26d93665e5bb

Reply:
OK ignore me and those commits, I've just noticed [this line](https://github.com/wellcomecollection/wellcomecollection.org/pull/10199/files/5f7bcc0be4a5c82a42a1024b85094b749b443e1b#diff-8435393952a67d31fe3d50e7c9e73d18ed1ce632f4f9ae6605956557acc31e97R61), silly me.